mondrian/src/main/java/mondrian/olap/fun/BuiltinFunTable.java

```diff
@@ -74,2075 +76,1888 @@
+      } );
+
+    // <Member>.NextMember
+    builder.define(
+      new FunDefBase(
+        "NextMember",
+        "Returns the next member in the level that contains a specified member.",
+        "pmm" ) {
+        @Override
+        public Calc compileCall( ResolvedFunCall call, ExpCompiler compiler ) {
+          final MemberCalc memberCalc =
+            compiler.compileMember( call.getArg( 0 ) );
+          return new AbstractMemberCalc( call, new Calc[] { memberCalc } ) {
+            public Member evaluateMember( Evaluator evaluator ) {
+              Member member = memberCalc.evaluateMember( evaluator );
+              return evaluator.getSchemaReader().getLeadMember(
+                member, 1 );
+            }
+          };
+        }
+      } );
+
+    builder.define( OpeningClosingPeriodFunDef.OpeningPeriodResolver );
+    builder.define( OpeningClosingPeriodFunDef.ClosingPeriodResolver );
+
+    builder.define( MemberOrderKeyFunDef.instance );
+
+    builder.define( ParallelPeriodFunDef.Resolver );
+
+    // <Member>.Parent
+    builder.define(
+      new FunDefBase(
+        "Parent",
+        "Returns the parent of a member.",
+        "pmm" ) {
+        @Override
+        public Calc compileCall( ResolvedFunCall call, ExpCompiler compiler ) {
+          final MemberCalc memberCalc =
+            compiler.compileMember( call.getArg( 0 ) );
+          return new AbstractMemberCalc( call, new Calc[] { memberCalc } ) {
+            public Member evaluateMember( Evaluator evaluator ) {
+              Member member = memberCalc.evaluateMember( evaluator );
+              return memberParent( evaluator, member );
+            }
+          };
+        }
+
+        Member memberParent( Evaluator evaluator, Member member ) {
+          Member parent =
+            evaluator.getSchemaReader().getMemberParent( member );
+          if ( parent == null ) {
+            parent = member.getHierarchy().getNullMember();
+          }
+          return parent;
+        }
+      } );
+
+    // <Member>.PrevMember
+    builder.define(
+      new FunDefBase(
+        "PrevMember",
+        "Returns the previous member in the level that contains a specified member.",
+        "pmm" ) {
+        @Override
+        public Calc compileCall( ResolvedFunCall call, ExpCompiler compiler ) {
+          final MemberCalc memberCalc =
+            compiler.compileMember( call.getArg( 0 ) );
+          return new AbstractMemberCalc( call, new Calc[] { memberCalc } ) {
+            public Member evaluateMember( Evaluator evaluator ) {
+              Member member = memberCalc.evaluateMember( evaluator );
+              return evaluator.getSchemaReader().getLeadMember(
+                member, -1 );
+            }
+          };
+        }
+      } );
+
+    builder.define( StrToMemberFunDef.INSTANCE );
+    builder.define( ValidMeasureFunDef.instance );
+
+    //
+    // NUMERIC FUNCTIONS
+    builder.define( AggregateFunDef.resolver );
+
+    // Obsolete??
```

Comment:
Is there a "easy way" to check this? If not just leave it.

Reply:
it's legacy code, I would prefer to leave it as is for now and revisit later